docs/ProfilerOptions.md

```diff
@@ -0,0 +1,113 @@
+| `status`           | `status`          | Prints profiling status: whether profiler is active and for how long.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| `meminfo`          | `meminfo`         | Prints used memory statistics.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `list`             | `list`            | Show the list of profiling events available for the target process specified with PID.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
Above options can be extracted in a separate group: Actions.

Reply:
Addressed in [e6282c8](https://github.com/async-profiler/async-profiler/pull/1029/commits/e6282c83bc0aa6007df310fd5e638e9d4d5ed6f1)